src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -150,8 +158,6 @@ public void BuildBasicApplicationReleaseProfiledAotWithoutDefaultProfile ()
 		[Category ("DotNetIgnore")] // Not currently working, see: https://github.com/dotnet/runtime/issues/56163
```

Comment:
Do we need to remove these now?

Reply:
The tests that mention https://github.com/dotnet/runtime/issues/56163 don't work yet. I think we need changes in dotnet/runtime to fix those.

Folder names like `BuildAotApplicationAndBundle AndÜmläüts_arm64-v8a_False_True` fail, and I saw the `<MonoAOTCompiler/>` task output characters that looked like the encoding was messed up.